src/messages/MessageBuilder.cpp

```diff
@@ -385,8 +385,26 @@ EmotePtr makeAutoModBadge()
         Url{"https://dashboard.twitch.tv/settings/moderation/automod"}});
 }
 
-EmotePtr makeSharedChatBadge(const QString &sourceName)
+EmotePtr makeSharedChatBadge(const QString &sourceName, const QString &url)
 {
+    if (!url.isEmpty())
+    {
+        QString modifiedUrl = url;
+
+        modifiedUrl.replace("300x300", "70x70");
```

Comment:
Twitch provides 64x64 as well as 28x28. We should use these to avoid scaling issues.

Reply:
do they? i tried putting 64x64 on my pfp and it gave me a 403 error

example:
<https://static-cdn.jtvnw.net/jtv_user_pictures/a577667e-2c71-4420-983d-9609aac088f2-profile_image-64x64.png>

Reply:
In the mini bit of digging I've done, it looks like 28x28 has always worked.

Switched to 28x28 in [`d3f6587` (#5760)](https://github.com/Chatterino/chatterino2/pull/5760/commits/d3f65871ec33358573584d63d46859c8c743305f)